docs/aerodromes/Class-D/nzdn.md

```diff
@@ -0,0 +1,116 @@
+| 03     | `BASOS #P` | Cat A to D             | **Preferred departure for `QN` and western-bound international traffic.**                                                                 |
+| 03     | `AKPIN #P` | Cat A to D             | **Preferred departure for `NV` traffic.**                                                                                                 |
+| 03     | `SOUTH #`  | Cat A to D             | To `OREXI` and `PORUT`. Non-RNAV.                                                                                                         |
+| 03     | `SW #`     | Cat A to D             | To `BIDEL`, `MEGUG` `MIPAK`, `PORUT` and `QN`. Non-RNAV.                                                                                  |
+
+## Visual Departures
+When either runway is in use, TDN may issue northern-bound aircraft a visual departure direct to `BIDEL` on pilot's request, traffic permitting. Alternatively, when Runway 21 is in use, TDN may issue northern-bound aircraft on the `BIDEL #Q`or `BIDEL #S` a visual departure to `ATGUB` or `IDLIK` on pilot's request ([Refer to AIP chart for `BIDEL #Q` and `BIDEL #S`](https://www.aip.net.nz/assets/AIP/Aerodrome-Charts/Dunedin-NZDN/NZDN_62.7_62.8.pdf)).
+
+!!! example "Visual Departure Direct to `BIDEL`"
+    **Dunedin Tower**: *"ANZ676, recleared visual deaprture direct `BIDEL`."*
+
+!!! example "Runway 21 Visual Departure"
+    **Dunedin Tower**: *"ANZ758M, recleared left visual deaprture direct `ATGUB` to rejoin the SID."*
+
+
+# Arrivals
+STH shall issue STARs to arrivals without coordination with TDN. The `MIPAK #C` arrival for runway 03 may be used when requested by the pilot. TDN may choose to coordinate with STH or amend the arrival on first contact.
+
+## Runway 21
+All arrivals terminate at `ESKAT`, which is the IAF for the RNAV and ILS approach. 
+
+### Missed Approach
+In the event of a missed approach, aicraft should be instructed to carry out the published missed approach procedure. For the ILS and NDB procedures, this will take the aircraft straight out to `BE` NDB holding at `A040`. For the RNAV procedure, it will take the aircraft straight out to `AKLAR` holding at `A040`. The aircraft can then be issued the approprite arrival procedure from `AKLAR` to commence the approach again.
+
+## Runway 03
+All arrivals terminate at `AKLAR`, which is the IAF for the RNAV and ILS approach.
+
+### Missed Approach
+In the event of a missed approach, aicraft should be instructed to carry out the published missed approach procedure. For the ILS and NDB procedures, this will take the aircraft straight out to `MI` NDB, then a right turn to `BE` NDB holding at `A040`. For the RNAV procedure, it will take the aircraft to `DABCA` holding at `A040`. The aircraft can then be issued the approprite arrival procedure from `DABCA` to commence the approach again.
```

Comment:
```suggestion
In the event of a missed approach, aircraft should be instructed to carry out the published missed approach procedure. For the ILS and NDB procedures, this will take the aircraft straight out to `MI` NDB, then a right turn to `BE` NDB holding at `A040`. For the RNAV procedure, it will take the aircraft to `DABCA` holding at `A040`. The aircraft can then be issued the appropriate arrival procedure from `DABCA` to commence the approach again.
```